crates/rc-box/src/lib.rs

```diff
@@ -634,6 +635,50 @@ then the data will be pinned in memory and unable to be moved."),
 
         #[cfg(feature = "std")]
         impl<T: ?Sized> UnwindSafe for $RcBox<T> where Box<T>: UnwindSafe {}
+
+        #[cfg(feature = "unsize")]
+        doc_comment! {
+            concat!("Unsizes a pointer using the `unsize` crate.
+# Usage
+
+```
+# use rc_box::*;
+use unsize::{Coercion, CoerceUnsize};
+
+let unique = ", stringify!($RcBox), "::new(|| 42u32);
+let unique:", stringify!($RcBox), r"<dyn Fn() -> u32> =
+    unique.unsize(Coercion::<_, dyn Fn() -> u32>::to_fn());
+
+let value = (*unique)();
+assert_eq!(value, 42);
+```
+
+Another common usage would be to create an `dyn Any`.
```

Comment:
nitpick: this should be either "create `dyn Any`" or "create a `dyn Any`", not "create an `dyn Any`" to fit the rest of pointer-utils' crates' documentation.